system-parachains/bridge-hubs/bridge-hub-kusama/src/lib.rs

```diff
@@ -128,6 +144,11 @@ pub type Migrations = (
 	cumulus_pallet_xcmp_queue::migration::v4::MigrationToV4<Runtime>,
 	// permanent
 	pallet_xcm::migration::MigrateToLatestXcmVersion<Runtime>,
+	snowbridge_pallet_system::migration::v0::InitializeOnUpgrade<
+		Runtime,
+		ConstU32<BRIDGE_HUB_ID>,
+		ConstU32<ASSET_HUB_ID>,
+	>,
```

Comment:
@claravanstaden Is this a one-time shot, or do we need it for every runtime upgrade? If it's the former, please move it before `// permanent`

Reply:
https://github.com/polkadot-fellows/runtimes/pull/130/commits/59b35027a6df62c6a6224714cd1a7f560dc96b26